src/modm/driver/display/ssd1306_impl.hpp

```diff
@@ -80,105 +119,61 @@ modm::Ssd1306<I2cMaster, Height>::writeDisplay()
 	RF_END_RETURN(this->wasTransactionSuccessful());
 }
 
-template < class I2cMaster, uint8_t Height >
-modm::ResumableResult<bool>
+template<class I2cMaster, uint8_t Height>
+inline modm::ResumableResult<bool>
 modm::Ssd1306<I2cMaster, Height>::setOrientation(glcd::Orientation orientation)
 {
-	RF_BEGIN();
-
-	if ( RF_CALL(writeCommand((orientation == glcd::Orientation::Landscape0) ?
-			Command::SetSegmentRemap127 : Command::SetSegmentRemap0)) )
+	if (orientation == glcd::Orientation::Landscape0)
+	{
+		commandBuffer[0] = HardwareConfigCommands::SegmentRemap127;
+		commandBuffer[1] = HardwareConfigCommands::ComOutputScanDirectionDecrement;
+	} else if (orientation == glcd::Orientation::Landscape180)
 	{
-		RF_RETURN_CALL(writeCommand((orientation == glcd::Orientation::Landscape0) ?
-				Command::SetComOutputScanDirectionDecrement : Command::SetComOutputScanDirectionIncrement));
+		commandBuffer[0] = HardwareConfigCommands::SegmentRemap0;
+		commandBuffer[1] = HardwareConfigCommands::ComOutputScanDirectionIncrement;
 	}
 
-	RF_END_RETURN(false);
+	return writeCommands(2);
```

Comment:
Same here: RFs without the macros should only be used for forward convenience functions without logic.